airflow/providers/amazon/aws/waiters/ecs.json

```diff
@@ -0,0 +1,81 @@
+{
```

Comment:
Love seeing this put to use.   I think it's so much cleaner in the implementation.  :+1: